wire-ios-data-model/Tests/Source/Model/Observer/UserObserverTests.swift

```diff
@@ -580,26 +580,27 @@ extension UserObserverTests {
         XCTAssertTrue(user.needsToAcknowledgeLegalHoldStatus)
     }
 
-    func testThatItNotifiesTheObserverOfLegalHoldStatusChange_Removed() {
-        // given
-        let user = ZMUser.selfUser(in: uiMOC)
-        user.acknowledgeLegalHoldStatus()
-
-        let legalHoldClient = UserClient.createMockLegalHoldSelfUserClient(in: uiMOC)
-
-        let modifier: (ZMUser) -> Void = { _ in
-            self.performPretendingUiMocIsSyncMoc {
-                legalHoldClient.deleteClientAndEndSession()
-            }
-        }
-
-        // when
-        self.checkThatItNotifiesTheObserverOfAChange(user,
-                                                     modifier: modifier,
-                                                     expectedChangedFields: [.legalHoldStatus, .isUnderLegalHold])
-
-        XCTAssertTrue(user.needsToAcknowledgeLegalHoldStatus)
-    }
+    // TODO: [jacob] re-enable
+//    func testThatItNotifiesTheObserverOfLegalHoldStatusChange_Removed() {
```

Comment:
Could you please disable the test via the `AllTests.xctestplan`?

Reply:
Done